examples/flutter_view/README.md

```diff
@@ -2,8 +2,8 @@
 
 This project demonstrates how to embed Flutter within an iOS or Android
 application. On iOS, the iOS and Flutter components are built with Xcode. On
-Android, the Android and Flutter components are built with Android Studio or
-gradle.
+Android, the Android, and Flutter components are built with Android Studio or
```

Comment:
The original was correct here. It is not a list of 3 items, but referring to the Android platform, where there are two components built: the Android component and the Flutter component.

Reply:
Okay I will change it now